driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -172,6 +172,20 @@
     <desc>Turn on the passing of water isotope fields through the coupler</desc>
   </entry>
 
+  <entry id="FLDS_POLAR">
+    <type>logical</type>
+    <valid_values>TRUE,FALSE</valid_values>
+    <default_value>FALSE</default_value>
+    <values match="last">
+      <value compset="_MPASO%IBPISMF_">TRUE</value>
+      <value compset="_MPASO%IBDISMF_">TRUE</value>
+      <value compset="_MPASSI%DIB_">TRUE</value>
```

Comment:
this logic is missing the case when DIB is not active, but DISMF or PISMF is active.

Reply:
Can you make an E3SM issue? I already merged this to master and have been in the process of blessing the expected NML and regular DIFFs (which are just due to a change in the field list)

Reply:
yes, will do